File: SDKs/amazonaws.com/cloudfront/2017-03-25/python/src/sdk/models/shared/toomanystreamingdistributions.py
import dataclasses
from typing import Optional


@dataclasses.dataclass
class TooManyStreamingDistributions:
    r"""TooManyStreamingDistributions
    Processing your request would cause you to exceed the maximum number of streaming distributions allowed.
    """
    
    message: Optional[str] = dataclasses.field(default=None)
    

File: SDKs/amazonaws.com/cloudfront/2017-03-25/java/lib/src/main/java/openapisdk/models/shared/TooManyStreamingDistributions.java
package openapisdk.models.shared;



/**
 * TooManyStreamingDistributions
 * Processing your request would cause you to exceed the maximum number of streaming distributions allowed.
**/
public class TooManyStreamingDistributions {
    public String message;
    public TooManyStreamingDistributions withMessage(String message) {
        this.message = message;
        return this;
    }
}